main.js

```diff
@@ -10,17 +10,106 @@ maxerr: 50, browser: true */
 define(function (require, exports, module) {
     "use strict";
 
+    // Load brackets dependencies
     var AppInit              = brackets.getModule("utils/AppInit"),
+        EditorManager        = brackets.getModule("editor/EditorManager"),
+        LiveDevelopment      = brackets.getModule("LiveDevelopment/LiveDevMultiBrowser"),
         LiveDevServerManager = brackets.getModule("LiveDevelopment/LiveDevServerManager"),
+        Menus                = brackets.getModule("command/Menus"),
         PreferencesManager   = brackets.getModule("preferences/PreferencesManager"),
         ProjectManager       = brackets.getModule("project/ProjectManager"),
+        Resizer              = brackets.getModule("utils/Resizer"),
+        UrlParams            = brackets.getModule("utils/UrlParams").UrlParams,
+        // Load nohost dependencies
         Browser              = require("lib/iframe-browser"),
-        LiveDevelopment      = brackets.getModule("LiveDevelopment/LiveDevMultiBrowser"),
-        PostMessageTransport = require("lib/PostMessageTransport"),
         Launcher             = require("lib/launcher").Launcher,
-        NoHostServer         = require("nohost/src/NoHostServer").NoHostServer;
+        NoHostServer         = require("nohost/src/NoHostServer").NoHostServer,
+        PostMessageTransport = require("lib/PostMessageTransport");
+
+    // Loads the initial html document
+    var defaultHTML = require("text!lib/default.html");
+
+    var codeMirror,
+        _server,
+        fs           = appshell.Filer.fs(),
+        params       = new UrlParams(),
+        parentWindow = window.parent;
+
+    // Force entry to if statments on line 262 of brackets.js to create
+    // a new project
+    PreferencesManager.setViewState("afterFirstLaunch", false);
+    params.remove("skipSampleProjectLoad");
+
+    /**
+     * By default this extension will remove top and side bars
+     */
+    function init(){
+        hide();
+        parentWindow.postMessage(JSON.stringify({
+            type: "bramble:loaded"
+        }), "*");
+    }
+
+    /**
+     * This function calls all the hide functions
+     */
+    function hide() {
```

Comment:
There'll probably be `require()` calls we don't need in this file that were there for HideUI-related functionality.